src/controller/sms/signup/signup_steps.js

```diff
@@ -0,0 +1,221 @@
+/* eslint-disable no-param-reassign */
+const message = require('../message')
+const saveUser = require('../../../user/save_user')
+const issues = require('../../../public/issues').issues
+const events = require('../schedule')
+const debug = require('debug')('sms')
+
+function notStarted(req, step) {
```

Comment:
I assume these methods will be extracted later, once we have multiple flows (not mad it's all here for now, just double checking)